gremlin/values.yaml

```diff
@@ -126,32 +126,33 @@ gremlin:
     #   Daemonset as well as any pod security resource that governs it. Capabilities that are required for specific
     #   attacks can be removed from this list if running such attacks are not desired.
     capabilities:
-      - KILL        # Required to run Process Killer attacks
-      - NET_ADMIN   # Required to run network attacks
-      - SYS_BOOT    # Required to run Shutdown attacks
-      - SYS_TIME    # Required to run Time Travel attacks
+      - KILL            # Required to run Process Killer attacks
+      - NET_ADMIN       # Required to run network attacks
+      - SYS_BOOT        # Required to run Shutdown attacks
+      - SYS_TIME        # Required to run Time Travel attacks
+      - DAC_READ_SEARCH # Required to run Certificate Expiry attacks, and dependency discovery features
```

Comment:
Gremlin has required this permission since `2.25.0` for service discovery features to work properly. Since most container runtimes provide DAC_OVERRIDE by default, issues stemming from its absence went unnoticed.